benchmark/http/check_invalid_header_char.js

```diff
@@ -1,6 +1,7 @@
 'use strict';
 
 const common = require('../common.js');
+// TODO: benchmark has access to internal modules?
```

Comment:
should the require be updated to `internal/http/common`? :slightly_smiling_face: